data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/main/ConsumerVerticleFactoryImpl.java

```diff
@@ -178,14 +172,15 @@ public AbstractVerticle get(final DataPlaneContract.Resource resource, final Dat
           Filter.noop();
 
         final var responseHandler = getNoopResponseHandlerOrDefault(egress, () -> getKafkaResponseHandler(vertx, producerConfigs, resource));
+        final var commitIntervalMs = Integer.parseInt(String.valueOf(consumerConfigs.get(ConsumerConfig.AUTO_COMMIT_INTERVAL_MS_CONFIG)));
```

Comment:
should we document this?

Reply:
Sure, I'll open an issue in knative/docs.

Reply:
:+1: